example/requirements.txt

```diff
@@ -0,0 +1,5 @@
+uvicorn
+starlette[full]
+aiokafka
+asyncio-redis
+asyncpg
```

Comment:
I guess we're missing `broadcaster` here.
Perhaps we should use `broadcaster[redis,postgres,kafka]`?